orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -573,3 +587,45 @@ def testInstantiateORTTrainer(step_fn, lr_scheduler, expected_lr_values):
     assert (trainer_from_onnx._onnx_model.graph == trainer._onnx_model.graph)
     assert (onnx.helper.printable_graph(trainer_from_onnx._onnx_model.graph) == onnx.helper.printable_graph(trainer._onnx_model.graph))
 
+
+@pytest.mark.parametrize("seeds, device_id", [
+    ((0, 0), 'cpu'),
```

Comment:
I would simplify by using just one seed unless there is a scenario we can say this two values can capture a bug